src/fabric-custom/test.json

```diff
@@ -3,5 +3,6 @@
   "ThirdPartyTag": "https://tpc.googlesyndication.com/pagead/imgad?id=CICAgKCrtKGjoAEQARgBMgjYagW3ARF6Fw",
   "TrackingPixel": "https://gu.com/tracking-pixel.gif",
   "ResearchPixel": "https://gu.com/research-pixel.gif",
-  "ViewabilityTracker": "<img src='https://gu.com/viewability-tracker.gif'>"
+  "ViewabilityTracker": "<img src='https://gu.com/viewability-tracker.gif'>",
+  "thirdPartyJSTracking": "<SCRIPT TYPE='application/javascript' SRC='https://pixel.adsafeprotected.com/rjss/st/726370/54949606/skeleton.js'></SCRIPT> <NOSCRIPT><IMG SRC='https://pixel.adsafeprotected.com/rfw/st/726370/54949605/skeleton.gif?gdpr=${GDPR}&gdpr_consent=${GDPR_CONSENT_278}&gdpr_pd=${GDPR_PD}' BORDER=0 WIDTH=1 HEIGHT=1 ALT=''></NOSCRIPT>"
```

Comment:
_1. why would you set HTML/JS content in JSON file, rather than directly in the template_

The JSON file is only for testing GAM variables locally in the preview, each field matches it's corresponding macro in the the template. The JSON file itself is not exported to GAM.

_2 how are the variables, like ${GDPR_PD}, going to be replaced when there is no JavaScript, as this is inside the noscript tag._

Yes, that's a good point on ${GDPR_PD} var in that tag. That was just a tracking tag from adops that I grabbed for testing . But yes looking at it I can see the problem. Went through a few more creatives and found a more typical example below. 

<script type="text/javascript" src="https://z.moatads.com/krmedialvmh68499063739/moatad.js#moatClientLevel2=4748378&moatClientLevel1=22803291&moatClientLevel3=249144050&moatClientLevel4=1x1_Site_Served"></script>

Reply:
> Hey @Onelittlewolfe, just bringing this request back to the fore.
> 
> I don’t think we should be using the `gu.com` domain any longer. I’m also not sure we want to allow third-party tracking on our native template. Could we set a meeting to discuss this a little further?

Hi @mxdvl sure happy to have a meeting, may need to have someone from Adops as well.